jetbrains-ultimate/src-201/software/aws/toolkits/jetbrains/services/redshift/auth/AwsAuthWidget.kt

```diff
@@ -0,0 +1,55 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.redshift.auth
+
+import com.intellij.database.dataSource.DataSourceUiUtil
+import com.intellij.database.dataSource.LocalDataSource
+import com.intellij.database.dataSource.url.template.ParametersHolder
+import com.intellij.database.dataSource.url.template.UrlEditorModel
+import com.intellij.database.dataSource.url.ui.UrlPropertiesPanel
+import com.intellij.ui.components.JBLabel
+import com.intellij.ui.components.JBTextField
+import software.aws.toolkits.jetbrains.services.redshift.RedshiftUtils
+import software.aws.toolkits.jetbrains.ui.AwsAuthWidget
+import software.aws.toolkits.resources.message
+import javax.swing.JPanel
+
+const val CLUSTER_ID_PROPERTY = "AWS.RedshiftClusterId"
+
+class AwsAuthWidget : AwsAuthWidget() {
```

Comment:
can we rename one of these?

Reply:
Yep, renamed in a different branch but forgot to pull it in, renamed